source/index.html

```diff
@@ -474,7 +474,20 @@ <h2 class="recipe-section-header">Description</h2>
         <hr class="recipe-horizontal-rule" />
 
         <!-- Recipe Ingredients -->
-        <h2 class="recipe-section-header">Ingredients Checklist</h2>
+        <div>
+          <h2 class="recipe-section-header">Ingredients Checklist</h2>
+        </div>
+
+        <div>
+          Serving scale:
```

Comment:
I think `Serving size` probably makes more sense and we should probably leave a disclaimer that adjusted amounts are estimates
```suggestion
          Serving size (scaled amounts are estimates):
```